src/pupil_labs/invisible_lsl_relay/cli.py

```diff
@@ -63,6 +64,13 @@ async def get_user_selected_device(self):
                 print()
                 print("To reload the list, hit enter. ")
                 print("To abort device selection, use 'ctrl+c' and hit 'enter'")
+                if self.n_reload >= 5:
+                    print(
+                        "Can't find the device you're looking for? \n"
+                        "Make sure the Companion App is connected to the same "
+                        "network and at least version v1.4.14. "
```

Comment:
Remove trailing whitespace
```suggestion
                        "Can't find the device you're looking for?\n"
                        "Make sure the Companion App is connected to the same "
                        "network and at least version v1.4.14."
```